integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/segment/ShowSegmentTestCase.scala

```diff
@@ -187,6 +187,16 @@ class ShowSegmentTestCase extends QueryTest with BeforeAndAfterAll {
     dropTable(tableName)
   }
 
+  test("test for load time and format name") {
```

Comment:
can you add one test case for file format info in show segments in addsegment test cases?

Reply:
added in AddSegmentTestCase